packages/mantine/src/components/table/Table.tsx

```diff
@@ -293,7 +289,6 @@ export const TableComponentsOrder = {
 };
 
 Table.AccordionColumn = TableAccordionColumn;
-Table.Actions = TableActions;
 Table.ActionsColumn = TableActionsColumn;
 Table.ActionItem = TableActionItem;
 Table.ActionLabel = Menu.Label;
```

Comment:
Do all components always have to be accessible?
```
Table.ActionsColumn = TableActionsColumn;
Table.ActionItem = TableActionItem;
Table.ActionLabel = Menu.Label;
Table.ActionDivider = Menu.Divider;
```

Reply:
Lines 308 and 309 are code duplication ;)

![image](https://github.com/coveo/plasma/assets/966550/8b3089f3-dfa5-48da-a0ac-77c650882e68)

Reply:
What do you mean? Are you talking about component like TableActionList? Those are internal components and shouldn't be used in the admin

Reply:
I see you've removed the `Table.Actions` component, but the others should still be present? :arrow_down: 

```
Table.ActionsColumn = TableActionsColumn;
Table.ActionItem = TableActionItem;
Table.ActionLabel = Menu.Label;
Table.ActionDivider = Menu.Divider;
```

Reply:
I kept `Table.ActionsColumn` and `Table.ActionItem`, I removed the others